mne/preprocessing/eog.py

```diff
@@ -132,23 +131,30 @@ def _find_eog_events(eog, event_id, l_freq, h_freq, sampling_rate, first_samp,
 
 
 def _get_eog_channel_index(ch_name, inst):
-    """Get EOG channel index."""
+    """Get EOG channel indices."""
+    # Ensure we have a list of channel names
     if isinstance(ch_name, str):
-        # Check if multiple EOG Channels
         if ',' in ch_name:
+            warn(f'You supplied multiple EOG channel names, separated by a '
+                 f'comma: {ch_name}. We will stop supporting this in version '
+                 f'0.24. Please pass a list of channels instead: '
+                 f'{ch_name.split(",")}', DeprecationWarning)
             ch_name = ch_name.split(',')
         else:
             ch_name = [ch_name]
+    elif ch_name is not None:
+        ch_name = list(ch_name)
 
+    if isinstance(ch_name, list):
         eog_inds = pick_channels(inst.ch_names, include=ch_name)
 
         if len(eog_inds) == 0:
             raise ValueError('%s not in channel list' % ch_name)
         else:
-            logger.info('Using channel %s as EOG channel%s' % (
-                        " and ".join(ch_name), _pl(eog_inds)))
-    elif ch_name is None:
+            logger.info(f'Using: {" and ".join(ch_name)} '
+                        f'as EOG channel{_pl(eog_inds)}')
 
+    elif ch_name is None:
```

Comment:
why `elif` here? We should be guaranteed by this point that it's a list, right? I would actually add a check above...

Reply:
No, if can still be "legally" `None`: this is allowed such that we can now start finding EOG channels via `pick_types()`. But as it seems to be confusing, I can try to refactor it

Reply:
Sorry yes it can be None but this should just be
```
else:
    assert ch_name is None
```
or so, i.e., no need for `elif`